app/controllers/staff_info_controller.rb

```diff
@@ -0,0 +1,80 @@
+class StaffInfoController < ApplicationController
+  before_action :authorize_prison_request
+
+  STAFF_INFO_ENDPOINT = ENV.fetch('STAFF_INFO_ENDPOINT', nil)
+
+  before_action :not_found, unless: :endpoint_set?
+  before_action :send_pdf, if: :pdf?
+  before_action :fetch_nav, only: :show
+  before_action :proxy_content, only: :show
+
+  def index
+    @content = proxy_content
+    render :index, status: req.status
+  end
+
+  def show
+    @content = markdown? ? convert_markdown : proxy_content
+    case req.status
+    when 200 then render :show
+    when 404 then render 'errors/404', status: 404
+    when 500 then render 'errors/500', status: 500
+    when 503 then render 'errors/503', status: 503
```

Comment:
I don't particularly see the purpose of proxying these statuses 1-1, and this code will render nothing in the case that the status is not in the list.

I would suggest that a non-200 error code simply raises an exception which is handled by the errors controller. I'd be happy with handling 5xx separately and returning an error, but I'm not fussed.